assets/src/components/notifications.tsx

```diff
@@ -43,37 +43,49 @@ export const NotificationCard = ({
   notification: Notification
   remove: (id: number) => void
   currentTime: Date
-}) => (
-  <div className="m-notifications__card">
-    <button className="m-notifications__card-info">
-      <div className="m-notification__title-row">
-        <div className="m-notification__title">
-          {title(notification.reason)}
+}) => {
+  const [isNew, setIsNew] = useState<boolean>(true)
+  useEffect(() => {
```

Comment:
@skyqrose perhaps `useLayoutEffect` (https://reactjs.org/docs/hooks-reference.html#uselayouteffect) would be more appropriate here. From the documentation:

> However, not all effects can be deferred. For example, a DOM mutation that is visible to the user must fire synchronously before the next paint so that the user does not perceive a visual inconsistency.

Reply:
That certainly would make sense, but I tried it, and it made it consistently not work. I think it's backwards from what we want, in that it forces DOM>effect>paint, rather than DOM>paint>effect. (We want the paint to be first so that the css transition sees the starting state.)

(Without the timeout, `useEffect` works about half the time.)